framework/src/engine/bft/index.ts

```diff
@@ -12,7 +12,9 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+export * from './constants';
 export { BFTModule } from './module';
 export type { BFTMethod } from './method';
+export * from './schemas';
```

Comment:
schemas file has a lot of interfaces defined within it, it would be best to avoid exposing all of them but just expose `bftParametersSchema`, `BFTParameters` and `Validator`

Reply:
`Validator` is not exportable from here